src/librustdoc/html/layout.rs

```diff
@@ -45,7 +46,8 @@ impl<'a> Page<'a> {
     }
 }
 
-#[derive(Serialize)]
+#[derive(Serialize, Template)]
```

Comment:
Maybe `Serialize` is unnecessary for `PageLayout`, `ItemVars` and their fields?

Reply:
Good catch! Cleaned those up locally.